scripts/verify-accessibility-results.js

```diff
@@ -0,0 +1,70 @@
+const { getAccessibilityResults } = require('@cypress/extract-cloud-results')
+
+/**
+ * The current problematic rules that need to be addressed.
+ * View the Accessibility report for the Cypress run in the Cloud
+ * for more details.
+ */
+const problematicRules = [
+  'aria-required-children',
+  'aria-required-parent',
+  'button-name',
+  'image-alt',
+  'label',
+  'aria-allowed-role',
+  'empty-heading',
+  'heading-order',
+  'page-has-heading-one',
+  'aria-dialog-name',
+  'aria-hidden-focus',
+  'aria-input-field-name',
+  'aria-toggle-field-name',
+  'color-contrast',
+  'link-in-text-block',
+  'list',
+  'listitem',
+  'nested-interactive',
+  'scrollable-region-focusable',
+]
+
+getAccessibilityResults({
```

Comment:
Talked with @emilyrohrbough and sounds like this is an implementation detail we don't want to comment here, but going to throw it here for posterity in the case we might need to revisit something
```suggestion
// the function polls every 30 seconds until the reports are done processing. This will poll up to 30 times
// (max 15 minutes) and if the reports are not done processing by then an error will throw.
// if processing fails for any report, the function will fail and not wait for other processing to occur.
getAccessibilityResults({
```

Reply:
@AtofStryker https://github.com/cypress-io/cypress/pull/30325/commits/9319a6754a968e0e8dfbe3b7c8256dd4f9039aa5